virtwho/config.py

```diff
@@ -685,29 +694,487 @@ def addConfig(self, config):
         self._configs.append(config)
 
 
-def getOptions(section, parser):
-    options = {}
-    for option in parser.options(section):
-        options[option] = parser.get(section, option)
-    return options
-
-
-def getSections(parser):
-    sections = {}
-    for section in parser.sections():
-        try:
-            sections[section] = getOptions(section, parser)
-        except NoOptionError:
-            sections[section] = {}
-    return sections
-
-
-def parseFile(filename, logger=None):
+def parseFile(filename):
     # Parse a file into a dict of section_name: options_dict
     # options_dict is a dict of option_name: value
     parser = StripQuotesConfigParser()
     fname = parser.read(filename)
     if len(fname) == 0 and logger:
         logger.error("Unable to read configuration file %s", filename)
-    sections = getSections(parser)
+    sections = parser._sections
+    for section in sections:
+        if '__name__' in sections[section]:
+            del sections[section]['__name__']
     return sections
+
+# Helper methods used to validate parameters given to virt-who
+def str_to_bool(value):
+    if isinstance(value, bool):
+        return value
+    if isinstance(value, str):
```

Comment:
To advocate importing `six` module you can use here: `if isinstance(value, six.string_types):